README.md

```diff
@@ -46,6 +49,34 @@ Prior vSphere 8 Update 1, the Supervisor Services are only available with Superv
 | External DNS Service - ExternalDNS | ❌ | ✅ |
 *\* The embedded Harbor Registry feature is still available and supported on vSphere 7 and onwards.*
 
+## Consumption Interface
+
+Provides the Local Consumption Interface (LCI) for Namespaces within vSphere Client. This also includes the Single Sign On (SSO) component required by the Cloud Consumption Interface (CCI) in Aria Automation within VMware Cloud Foundation.
+
+The minimum required version for using this interface is vSphere 8 Update 3.
+
+### Consumption Interface Versions
+
+Installation instructions can be found [here in VMware documentation](https://docs.vmware.com/en/VMware-vSphere/8.0/vsphere-with-tanzu-services-workloads/GUID-4843E6C6-747E-43B1-AC55-8F02299CC10E.html).
+
+NOTE: Occasionally, the plug-in might fail to load on the initial
```

Comment:
Isn't this a 'service' to the end user or does plugin means a vsphere plugin ?

Reply:
How about

NOTE: Occasionally, the supervisor service may fail to install the LCI vSphere plug-in on the initial
attempt. To check if the plug-in has loaded correctly, click the **vSphere Client**
menu icon, then to **Administration** -> **Client** -> **Plug-ins**.
Check the Status column of the *Namespace UI* plug-in, and in case you see a "Plug-in
configuration with Reverse Proxy failed." message, reinstall the plug-in.

Reply:
Yeah, looks good.